apps/parse/lib/parse/simple.ex

```diff
@@ -36,7 +38,13 @@ defmodule Parse.Simple do
 
   def parse(blob, row_callback) do
     blob
-    |> ExCsv.parse!(headings: true)
+    |> String.trim()
+    |> CSV.parse_string(skip_headers: false)
+    |> Stream.transform(nil, fn
```

Comment:
The trim is to address some cases where whitespace results in a blank row being used as a header row. 

The main logic here is basically just this: https://jacobswanner.com/development/2022/nimble-csv-maps/